cc/utils/cose/cose.h

```diff
@@ -81,10 +84,18 @@ class CoseKey {
   static absl::StatusOr<CoseKey> DeserializeHpkePublicKey(absl::string_view data);
   static absl::StatusOr<CoseKey> DeserializeHpkePublicKey(const std::vector<uint8_t>& data);
 
+  // Transforms HPKE public key into a COSE_Key and serializes it.
+  // TODO(#4818): This function is currently used for tests only. We need to
+  // refactor COSE classes to support both serialization and deserialization.
+  static absl::StatusOr<std::vector<uint8_t>> SerializeHpkePublicKey(
+      const std::vector<uint8_t>& public_key);
+
   const std::vector<uint8_t>& GetPublicKey() const { return x->value(); }
 
  private:
-  enum Parameter : int {
+  // COSE Key Common parameters.
+  // <https://datatracker.ietf.org/doc/html/rfc8152#section-7.1>
+  enum CoseKeyCommonParameter : int {
```

Comment:
partially pre-existing, but I think that in general using `enum class` is preferred. See https://abseil.io/tips/86

Reply:
I'm going to refactor this library as part of https://github.com/project-oak/oak/issues/4818

And will implement enum classes